plugins/modules/cronvar.py

```diff
@@ -218,8 +218,8 @@ def remove_variable_file(self):
         except OSError:
             # cron file does not exist
             return False
-        except Exception:
-            raise CronVarError("Unexpected error:", sys.exc_info()[0])
+        except Exception as e:
+            raise CronVarError("Unexpected error:", sys.exc_info()[0]) from e
```

Comment:
Nit picking, but this could probably be more readable as:

```suggestion
            raise CronVarError(f"Unexpected error: {e}") from e
```
technically the content is not exactly the same, but arguably str(e) will provide more useful information than just the exception class name. Same for the other change above.

Maybe for another PR, but just saying.

Reply:
Or remove that argument entirely - it looks like it is serving the exact same purpose of  `from e`.